portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,35 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+
+
+
+//Funtion for page tabs
+function openInfo(evt, info){
+    //Declare vars
+    var i, tabcontent, tablinks;
```

Comment:
You technically don't need to declare vars ahead of time. You could just declare and initialize in one single line, like `var tabcontent = document.getElementsByClassName("tabcontent");`. But this is more of a style preference, so it's up to you :)